src/stockwatch/entities.py

```diff
@@ -24,6 +25,74 @@ class ShareTransactionKind(Enum):
     DIVIDEND = auto()
 
 
+@dataclass(frozen=False, order=True)
+class CurrencyExchange:  # pylint: disable=too-many-instance-attributes
+    """For representing an exchange at a certain date.
+
+    Attributes
+    ----------
+    exchange_datetime     : the date for which the exchange is registered
```

Comment:
Docstring updated. I am not entirely sure if there will never be multipe exchange rows for a single date. And I want to be sure that we sort appropriately, otherwise the matching of a transaction to an exchange will probably go wrong. So therefore I use datetime rather than date. For the same reason, we have ShareTransaction.transaction_datetime rather than date.